### ISSUE TITLE
www/hs-yesod-bin: Broken by the upgrade

### ISSUE THREAD

**Issue by arrowd** [freebsd/freebsd-ports-haskell#19]
```
[1 of 1] Compiling Text.ProjectTemplate ( Text/ProjectTemplate.hs, dist/build/Text/ProjectTemplate.o, dist/build/Text/ProjectTemplate.dyn_o )

Text/ProjectTemplate.hs:60:21: error:
    Not in scope: `B64.joinWith'
    Module `Data.ByteString.Base64' does not export `joinWith'.
   |
60 |             yield $ B64.joinWith "\n" 76 $ B64.encode bs
   |                     ^^^^^^^^^^^^
```

Blocked by https://github.com/fpco/haskell-ide/issues/4
